kyaml/yaml/merge3/scalar_test.go

```diff
@@ -27,6 +27,57 @@ apiVersion: apps/v1`,
 apiVersion: apps/v1
 kind: StatefulSet # new value`},
 
+	//
+	// Test Case
+	//
+	{description: `Ensure comments are updated`,
```

Comment:
nit: Can we fix the formatting here? In pretty much every other test in the project we put `description` on a new line.